src/Components/Hero/hero.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from "@emotion/styled";
+import Box from "@mui/material/Box";
+import Container from "@mui/material/Container";
+import img1 from "../../Assets/mask.png";
+import Button from "../Button";
+import Nav from "../NavBar/Navbar";
+
+export const BoxContent = styled(Box)`
+  width: 35%;
+  display: flex;
+  align-items: start;
+  justify-content: center;
+  flex-direction: column;
+  gap: 30px;
+  margin-left: 25px;
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  font-weight: 200;
+  line-height: 30px;
+
+  h3 {
+    font-family: "Inter", sans-serif;
+    font-weight: 200;
+  }
+
+  span {
+    color: #d1094b;
+  }
+  h1,
+  br {
+    font-size: 60px;
+    font-weight: 500;
+    margin: 0;
+    padding: 0;
+    line-height: 80%;
+    letter-spacing: 0.5px;
+  }
+`;
+
+function Hero() {
+  return (
+    <Container
+      maxWidth="xl"
+      sx={{
+        backgroundImage: `url(${img1})`,
+        height: "100vh",
+        backgroundSize: "cover",
+        backgroundPosition: "center center",
+        display: "flex",
+        flexDirection: "column",
```

Comment:
try to use styled components for styling

Reply:
What do you mean by that I am using a styled component?